tox.ini

```diff
@@ -0,0 +1,36 @@
+[tox]
+description = Default tox environments list
+envlist =
+    style,{py37,py38,py39,py310}{,-coverage},doc
+skip_missing_interpreters = true
+isolated_build = true
+isolated_build_env = build
+
+[testenv]
+description = Checks for project unit tests and coverage (if desired)
+basepython =
+    py37: python3.7
+    py38: python3.8
+    py39: python3.9
+    py: python3
+    {style,reformat,doc}: python3
+setenv =
+    PYTHONUNBUFFERED = yes
+    coverage: PYTEST_EXTRA_ARGS = --cov=ansys.mapdl --cov-report=term --cov-report=xml:.cov/xml --cov-report=html:.cov/html
+extras = tests
+commands =
+    pytest {env:PYTEST_MARKERS:} {env:PYTEST_EXTRA_ARGS:} {posargs:-vv}
+
+[testenv:style]
+description = Checks project code style
+deps =
+    pre-commit
+commands =
+    pre-commit install
+    pre-commit run --all-files --show-diff-on-failure
+
+[testenv:doc]
+description = Check if documentation generates properly
+extras = doc
+commands =
+    sphinx-build -d "{toxworkdir}/doc_doctree" doc/source "{toxworkdir}/doc_out" --color -vW -bhtml
```

Comment:
in PyMAPDL, the sphinx-build is in ``doc\make``

The options I believe are different than the one specified here.

Reply:
We should look into that @germa89 - I know PyMAPDL has been around for long but you should try to align to the other PyAnsys repositories files/calls 😄 just so that all of them look the same. We can discuss this.

Reply:
Disregarding comment - it should work after investigation.